### ISSUE TITLE
[BUG] npm install gives error code 127

### ISSUE THREAD

**Issue by souvikinator** [khaosdoctor/devto-nodejs-sdk#15]
**Describe the bug**
on installing package from npm using `npm i devto-nodejs-sdk` throw 127 error with following message:
```
npm ERR! command sh -c ./node_modules/.bin/gitmoji -i
npm ERR! sh: 1: ./node_modules/.bin/gitmoji: not found
```
tried using another package (`@dendronhq/devto-nodejs-sdk`) which links to same repo and it works find but all the closed issues still persist.

**Comment by khaosdoctor** [comment]
Thanks for the report! I'll take a quick look, I think this can be removed, it should be a dev dependency.

**Comment by khaosdoctor** [comment]
Fixed by a5be131766222cabb0d409af18f3fcc03a40f1f2

Thanks for the contribution :) I'll publish a new patch

**Comment by souvikinator** [comment]
Awesome! Thanks for the fix
